img_conv_core.php

```diff
@@ -128,34 +127,35 @@ function convert($cf, $alpha = 0) {
                 $this->conv_px($x, $y);
             }
         }
-        
-        imagecopy ($this->img, $img_tmp,  0 , 0 , 0 , 0 , $this->w , $this->h);
+
+        // TODO !WARNING SUPPRESSED!
```

Comment:
This warning worries me the most. In some cases Null is copied into $this->img. Maybe it should be surrounded with if statement?